pyproject.toml

```diff
@@ -20,7 +20,6 @@ exclude =  ["env-example"]
 python = "^3.9,<3.13"
 geopandas = { version = "^0.13.2", optional = true }
 httpx = "^0.25.0"
-pandas = { version = "^2.1.1", optional = true }
```

Comment:
This mistake I will resolve in the next commit...

Reply:
Is there a reason not to add it back where it was? Packages were ordered alphabetically, but now they are not.